src/components/ContentHeader.vue

```diff
@@ -1,72 +1,62 @@
 <template>
   <div>
-    <div class="usa-overlay"></div>
 <header class="usa-header usa-header--basic">
   <div class="usa-nav-container">
-    <div class="usa-navbar">
-      <div class="usa-logo" id="basic-logo">
-        <em class="usa-logo__text">
-          <a href="javascript:void(0)" title="USGS Vizlab">
-            USGS Vizlab
-          </a>
-        </em>
-      </div>
-      <button class="usa-menu-btn">Menu</button>
-    </div>
     <nav aria-label="Primary navigation" class="usa-nav">
-      <button class="usa-nav__close">
-        <img src="/assets/img/usa-icons/close.svg" role="img" alt="Close" />
-      </button>
       <ul class="usa-nav__primary usa-accordion">
+         <li class="usa-nav__primary-item">
+          <button
+            class="usa-accordion__button usa-nav__link"
+            aria-expanded="false"
+            aria-controls="basic-nav-new"
+            onClick="document.getElementById('viz-new').scrollIntoView();"
+          >
+            <span>What's new</span>
+          </button>
+        </li>
         <li class="usa-nav__primary-item">
           <button
-            class="usa-accordion__button usa-nav__link usa-current"
+            class="usa-nav__button usa-nav__link usa-current"
             aria-expanded="false"
-            aria-controls="basic-nav-section-one"
+            aria-controls="basic-nav-viz"
+            onClick="document.getElementById('viz-cards').scrollIntoView();"
```

Comment:
Ok, so now that I realized `scroll-margin-left` was only working on Firefox, I did get this one fixed by directing to scroll to a div within the `viz-cards` section, rather than that section --- `onClick="document.getElementById('portfolio-accordions').scrollIntoView();"`

Reply:
Ok you were onto something here, I resolved the issue. I was applying the outer margins to the div that contained the entire grid. Because of that, when it scrolled to new sections, it was also horizontally scrolling to the leftmost edge of the div. Setting the margin to `<Section>` in `Visualization.vue` fixes it.